tableau_utilities/tableau_file/tableau_file.py

```diff
@@ -52,15 +55,21 @@ def __extract_xml(self, path=None):
             self._tree = ET.parse(path)
             self._root = self._tree.getroot()
 
-    def unzip(self, unzip_all=False):
+    def unzip(self, unzip_all=False, extract_to=None):
```

Comment:
@JustinGrilli  adding this parameter let's me get rid of that `save_xml` code I had and use your function.  I needed to be able to specific a new directory for the target  for the cli to work the way I wanted.